pkg/scheduler/apis/config/v1/default_plugins.go

```diff
@@ -53,6 +55,10 @@ func getDefaultPlugins() *v1.Plugins {
 		},
 	}
 
+	if utilfeature.DefaultFeatureGate.Enabled(features.DynamicResourceAllocation) {
+		plugins.MultiPoint.Enabled = append(plugins.MultiPoint.Enabled, v1.Plugin{Name: names.DynamicResources})
```

Comment:
At some point during our iterations in ComponentConfig, we decided that it doesn't make much sense to based the defaults on the feature gate. Yes, we did it differently in v1beta2 and before.

Instead, we check the feature gate in the plugin implementation.
The reason is that a user could still set a ComponentConfig that has the plugin enabled. Disabling the feature gate should disable the functionality even if the plugin is enabled.

Reply:
Changed to "unconditional".

Reply:
.. and added a feature gate check to the plugin.

Reply:
There is one test for the v1 config here: https://github.com/kubernetes/kubernetes/blob/b4156ea47b3caf0088c6316ecd89afe715830807/cmd/kube-scheduler/app/server_test.go#L85-L121

Should I update that to include "DynamicResources"?